src/cryptography/hazmat/primitives/asymmetric/ec.py

```diff
@@ -11,6 +11,58 @@
 from cryptography import utils
 
 
+class EllipticCurveField(object):
+    """
+    Field type of an EllipticCurve
+    """
+
+
+class EllipticCurvePrimeField(EllipticCurveField):
```

Comment:
Why are we using inheritance here? In general we've avoided that for every other public API surface.